server/http_client.go

```diff
@@ -0,0 +1,40 @@
+package server
+
+import (
+	"bytes"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type HttpClient interface {
```

Comment:
perhaps `RPCProxyClient` would be better?

Reply:
Updated